src/Support/Http/routes.php

```diff
@@ -4,14 +4,11 @@
     $routeName = config('lighthouse.route_name', 'graphql');
     $controller = config('lighthouse.controller');
 
-    if (config('lighthouse.route_enable_get', false)) {
-        app('router')->get($routeName, [
-            'as' => 'lighthouse.graphql',
-            'uses' => $controller,
-        ]);
-    }
+    $methods = config('lighthouse.route_enable_get', false) ?
```

Comment:
We do ternarys with a leading operator.

Reply:
Changed in https://github.com/nuwave/lighthouse/pull/738/commits/472f62a7529e7a93a5d0adae7b33e73ec81da8e1